src/acl/external/time_quota/ext_time_quota_acl.cc

```diff
@@ -26,18 +26,14 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111, USA.
  */
 
+/* DEBUG: section 94    Helpers */
```

Comment:
Please use existing section 82 for ease of integration with squid.conf.
```suggestion
/* DEBUG: section 82    External ACL Helpers */
```